### PR TITLE
Updated Laravel 5 project setup instructions

### PR DESCRIPTION
**Pull request by mmhyamin**

### CODE REVIEW

File: start/sw/web-app/laravel5.md [hunk 1]
```diff
@@ -7,74 +7,76 @@ order: 3
...
+sudo mysql_secure_installation
 ```
-#### Install PHP
-And extras
+
+#### Install PHP and Extensions
 ```bash
-# Install php cli and needed extras
-$ sudo dnf install php-cli php-mcrypt-5.6.20 php-mysqlnd
+# Enable remi repository for PHP 7
```
Comment (asamalik) [inline start/sw/web-app/laravel5.md]: Could you please tell me why do you need to enable an external repo? Is it something optional? If it's optional, it should be marked that way.

File: start/sw/web-app/laravel5.md [hunk 2]
```diff
@@ -7,74 +7,76 @@ order: 3
...
+# Setup Apache to automatically start upon system boot
+sudo systemctl enable httpd.service
+# Then to boot the server
+sudo systemctl start httpd
+# And stop with
+sudo systemctl stop httpd
+```
 
-#### Install MariaDB
+#### Install MariaDB Relational Database Server
```
Comment (strzibny) [inline start/sw/web-app/laravel5.md]: Do we want to duplicate the content here? We should rather link (and improve) https://developer.fedoraproject.org/tech/database/mariadb/about.html.

File: start/sw/web-app/laravel5.md [hunk 3]
```diff
@@ -7,74 +7,76 @@ order: 3
...
 ```
-#### Install PHP
-And extras
+
+#### Install PHP and Extensions
 ```bash
-# Install php cli and needed extras
-$ sudo dnf install php-cli php-mcrypt-5.6.20 php-mysqlnd
+# Enable remi repository for PHP 7
+wget http://rpms.remirepo.net/fedora/remi-release-24.rpm
```
Comment (strzibny) [inline start/sw/web-app/laravel5.md]: I agree with @asamalik. Actually would be better to link Copr repo if this have to be outside the official one. Don't know why the php 7 is not present here: https://copr.fedorainfracloud.org/coprs/remi/.

File: start/sw/web-app/laravel5.md [hunk 4]
```diff
@@ -7,91 +7,86 @@ order: 3
...
+$ sudo systemctl enable httpd.service
+# Then to boot the server
+$ sudo systemctl start httpd
 # And stop with
-$ sudo systemctl stop mariadb.service
+$ sudo systemctl stop httpd
 ```
+
+#### Install MariaDB Relational Database Server
+Follow the MariaDB installation instructions [here](../../../tech/database/mariadb/about.html).
```
Comment (strzibny) [inline start/sw/web-app/laravel5.md]: Please look here on how make internal links: https://developer-portal.github.io/contributing/write-introduction.html

e.g. `/tech/..`

File: start/sw/web-app/laravel5.md [hunk 5]
```diff
@@ -7,91 +7,86 @@ order: 3
 ## Laravel 5
 > The PHP Framework For Web Artisans
 
-Laravel is a high-level PHP Web framework 
+Laravel is a high-level PHP Web framework. We have to make sure all of these requirements are satisfied.
```
Comment (strzibny) [inline start/sw/web-app/laravel5.md]: I would reword the second sentence as "To use (or install) Laravel the following needs to be satisfied:"

The sentence as it is now is somehow out of context.

File: start/sw/web-app/laravel5.md (hunk 4 above)
Comment (mmhyamin) [inline start/sw/web-app/laravel5.md]: Done

File: start/sw/web-app/laravel5.md (hunk 5 above)
Comment (mmhyamin) [inline start/sw/web-app/laravel5.md]: Done.

File: start/sw/web-app/laravel5.md [hunk 6]
```diff
@@ -7,91 +7,86 @@ order: 3
...
-$ sudo dnf install php-cli php-mcrypt-5.6.20 php-mysqlnd
+# Laravel required PHP extensions.
+$ sudo dnf install php php-common php-cli php-pdo php-mbstring php-zip php-xml
+# Restart Apache
+$ sudo systemctl restart httpd
 ```
-Now we will need composer to install laravel and it's dependancies 
+
+#### Install Composer
+We will need composer to install laravel and it's dependancies
```
Comment (strzibny) [inline start/sw/web-app/laravel5.md]: dependEncies
